### PR TITLE
Assorted packaging improvements

### PR DESCRIPTION
**Pull request by jwodder**

### CODE REVIEW

**Comment by jwodder** [comment]
@asmacdo Ready for review.

File: MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1,5 @@
+include CHANGELOG.* CONTRIBUTORS.* LICENSE tox.ini
+graft src
+graft test
+include test_script.py
```
Comment (asmacdo) [inline MANIFEST.in:4]: @jwodder why include tests in the sdist?

**Review by asmacdo — APPROVED** [review]
LGTM, just a non-blocking question

File: MANIFEST.in (hunk 1 above)
Comment (jwodder) [reply MANIFEST.in:4]: So that repackagers who build off the sdist can run the tests.

**Pull request merged**